src/trie/merkle_tree.rs

```diff
@@ -699,6 +730,160 @@ impl<H: StarkHash, DB: BonsaiDatabase, ID: Id> MerkleTree<H, DB, ID> {
     /// preload_nodes from the current root towards the destination [Leaf](Node::Leaf) node.
     /// Returns the list of nodes along the path.
     ///
+    /// if it exists, or down to the node which proves that the key does not exist.
```

Comment:
The doc above this line belongs to the function `preload_nodes` and should be moved back there

Reply:
done